spot-contracts/contracts/FeePolicy.sol

```diff
@@ -376,6 +387,12 @@ contract FeePolicy is IFeePolicy, OwnableUpgradeable {
 
     /// @inheritdoc IFeePolicy
     function computeRebalanceData(SubscriptionParams memory s) external view override returns (RebalanceData memory r) {
+        // We skip rebalancing if dr is close to 1.0
+        uint256 dr = computeDeviationRatio(s);
+        if (dr > rebalEqDr.lower && dr < rebalEqDr.upper) {
```

Comment:
```suggestion
        if (dr >= rebalEqDr.lower && dr <= rebalEqDr.upper) {
```
Shouldn't really matter in practice, but I find it easier to think of this as a closed interval.